scss/components/_contact-cases.scss

```diff
@@ -62,6 +81,92 @@
   }
 }
 
+.civicase__contact-cases-details {
```

Comment:
Should be moved to a new file.

Reply:
It is a part of the contact cases tab. I am anxious to know why do you think this should be a new css file.